Dockerfile

```
@@ -0,0 +1,52 @@
+ARG PY_VERSION=3.9
+
+FROM amazon/aws-lambda-python:$PY_VERSION
+
+# Declare it a second time so it's brought into this scope.
+ARG PY_VERSION=3.9
+# Get the base output file name from the command line if provided.
+ARG FILE_NAME=example
+
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+# Note: Additional labels are added by the build workflow.
+LABEL org.opencontainers.image.authors="nicholas.mcdonnell@cisa.dhs.gov"
+LABEL org.opencontainers.image.vendor="Cyber and Infrastructure Security Agency"
```

As goofy as it is, this is our official agency name:
```suggestion
LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
```
This also matches the vendor label that we have in [`cisagov/skeleton-docker`](https://github.com/cisagov/skeleton-docker/blob/46732472c238367e5b21bea88f9081a6a7cef7aa/Dockerfile#L11).

---

Yep I copied stuff from [here](https://github.com/cisagov/skeleton-aws-lambda/blob/4e041e53e205429bbbe5d96b0c9d51d93e9a2dc8/Dockerfile#L15) and didn't notice that was missed. Thanks for catching it!